.github/workflows/integration_tests.yml

```diff
@@ -49,9 +49,9 @@ jobs:
         env:
           FAL_STATS_ENABLED: false
         run: |
+          BEHAVE_ARGS=""
```

Comment:
ATL: Any benefit of this change. I find the old version neater.

Reply:
I wanted to test only certain `.feature` and this enabled me to change it in a single place instead of two lines.

Reply:
I actually made it not as useful when removing the `.feature` thing. Fixed it to be more useful.